runtime/compiler/control/CompilationThread.cpp

```diff
@@ -7519,7 +7519,18 @@ TR::CompilationInfoPerThreadBase::preCompilationTasks(J9VMThread * vmThread,
          }
       }
 #if defined(J9VM_OPT_JITSERVER)
-   bool cannotDoRemoteCompilation = cannotPerformRemoteComp(
+   TR_Debug *debug = TR::Options::getDebug();
+   bool dontRemoteCompile = false;
+   if(debug)
+   {
+      setCompilationShouldBeInterrupted(0); // zero the flag because createResolvedMethod can
+					   // acquire/releaseVMaccessIfNeeded and may see the flag set by previous compilation
+      TR_ResolvedMethod *resolvedMethod = fe->createResolvedMethod(&trMemory, (TR_OpaqueMethodBlock *)method);
+      dontRemoteCompile = !_compInfo.methodCanBeRemotelyCompiled(resolvedMethod->signature(&trMemory),resolvedMethod->convertToMethod()->methodType());
+      if (TR::Options::getVerboseOption(TR_VerboseJITServer) && dontRemoteCompile)
```

Comment:
I don't think we need the verbose log. If you specify the exclude option I would image you shouldn't question why the compilation is local.